test/xml_namespace_test.rb

```diff
@@ -122,34 +124,23 @@ class Library < Representable::Decorator
       end
 
       collection :character, class: Model::Character do
-        namespace "http://eric.van-der-vlist.com/ns/library"
+        namespace "http://eric.van-der-vlist.com/ns/person"
 
         property :qualification
 
-        property :name, namespace: "http://eric.van-der-vlist.com/ns/person"
```

Comment:
Why change this line ? Shouldn't we check both `namespace` as method and `namespace` as option ?

Reply:
Namespace already inherited from parent object.

It the same

Reply:
No, I think the namespace defined at property level overrides the parent one. For example, `character.qualification` remains `lib` where as `character.name` changes to `hr`

Reply:
Ok, I think let's skip this for jruby and we'll come up with different example so it can be compatible to test overriding the namespace.